server/routerlicious/packages/lambdas-driver/src/document-router/documentPartition.ts

```diff
@@ -201,4 +203,43 @@ export class DocumentPartition {
 		this.activityTimeoutTime =
 			activityTime !== undefined ? activityTime : cacluatedActivityTimeout;
 	}
+
+	public pause(offset: number) {
+		if (this.paused) {
+			Lumberjack.info("Doc partition already paused, returning early.", {
+				documentId: this.documentId,
+				tenantId: this.tenantId,
```

Comment:
Would it be possible to also log the `offset` param along with the currently paused partition's offset?
I doubt this "already paused" case will happen often, but if it does, it seems that it could be possible for odd interactions where the next pause has a higher offset or something. In that case, it would be nice to have a bit of extra telemetry data ready to go.